romtools/vector_space/__init__.py

```diff
@@ -61,13 +61,13 @@
 $\\mathbf{u}_{\\mathrm{shift}} \\in \\mathbb{R}^{ N_{\\mathrm{vars}}  N_{\\mathrm{x}}}$ is the shift vector (or affine offset),
 and, by definition, $\\mathcal{V} \\equiv \\mathrm{range}(\\boldsymbol \\Phi)$.
 
-The `TrialSpace` abstract class defined below encapsulates the information of an affine trial space, $\\mathcal{V}$,
-by virtue of providing access to a basis matrix, a shift vector, and the dimensionality of the trial space,
+The `VectorSpace` abstract class defined below encapsulates the information of an affine vector space, $\\mathcal{V}$,
+by virtue of providing access to a basis matrix, a shift vector, and the dimensionality of the vector space,
```

Comment:
Do we want to keep "basis **matrix**" and "shift **vector**" here?

Reply:
i think we shuld do in a separate PR

Reply:
A separate PR is fine by me for this.